### ISSUE TITLE
Tests won't compile because library is published as ES6

### ISSUE THREAD

**Issue by henrikra** [callstack/react-native-testing-library#49]
I am trying to run simple test case in TypeScript project but I am getting 

```bash
> mocha
/myApp/node_modules/react-native-testing-library/src/index.js:2
import render from './render';
^^^^^^
SyntaxError: Unexpected token import
```

This means that you guys are publishing the project code in ES6 even though node_modules should be released as ES5.

Could you compile the library code to ES5 and republish that :)

**Comment by sibelius** [comment]
You should ask Babel-jest to transpile this library instead

**Comment by henrikra** [comment]
I am using mocha. I think the library code should be available as possible like other libraries are doing. If you really really really want to have library code in ES6 then there should be some steps on the README on how to run it. But that doesn't make any sense since that would require different guides to all different tests runners

**Comment by thymikee** [comment]
We mainly target React Native, where projects are released as source. Before you feed this lib to your TS compiler, you'll need to run it through Babel with `metro-react-native-babel-preset`.

I think we could compile it to ES5 though, I'm happy to review and accept the PR with that change :)

**Comment by thymikee** [comment]
Feel free to fix it, this is an open source project and I didn't say no to this kind of improvement 😃.

**Comment by henrikra** [comment]
@thymikee Okay do you have something in mind that you would use to compile this to ES5?

**Comment by thymikee** [comment]
Yup, I'd go with `@babel/preset-env` with a target of Node 8 (that's the minimum required by React Native right now).

**Comment by thymikee** [comment]
Or actually we could just go with `metro-react-native-babel-preset`, already available in dev deps. I think that will be simpler to set up

**Comment by henrikra** [comment]
So should the compilation be in `release-it`? Looks like you guys are using that but I don't have experience with it

**Comment by thymikee** [comment]
Don't worry about `release-it`. There should just be a git-ignored (but not npm-ignored) `build` folder with compiled sources, that would get created/updated on pre-release npm hook.

**Comment by Jukkish** [comment]
@thymikee I made a PR based on your tips, can you check it out?

**Comment by thymikee** [comment]
Just published `@1.4.1-es5.0` with compiled sources. Let me know if it works for you, both TS and Flow users

**Comment by thymikee** [comment]
@henrikra @Jukkish ping ^
